### PR TITLE
Bmw benchmark

### PR DESCRIPTION
**Pull request by jiivan**
Change blender benchmark to more intensive BMW benchmark from https://www.blender.org/download/demo-files/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ Golem communicates with external technologies some of them may be downloaded and
 
 Benchmarks:
 * General: [Minilight](http://www.hxa.name/minilight) by Harrison Ainsworth / HXA7241 and Juraj Sukop
-* Blender: [scene-Helicopter](https://www.blender.org/download/demo-files/)
+* Blender: [scene-Helicopter/BMW](https://www.blender.org/download/demo-files/)
```
Comment (badb) [inline README.md]: Just BMW

File: tests/golem/docker/test_blender_job.py [hunk 2]
```diff
@@ -36,29 +33,26 @@ def test_blender_job(self):
...
+        scene_file /= "apps/blender/benchmark/test_task/cube.blend"
+        shutil.copy(str(scene_file), self.resources_dir)
+        dest_scene_file = pathlib.Path(DockerJob.RESOURCES_DIR)
+        dest_scene_file /= scene_file.name
 
         params = {
             "outfilebasename": "out",
-            "scene_file": DockerJob.RESOURCES_DIR + "/" +
-                          path.basename(scene_files[0]),
+            "scene_file": str(dest_scene_file),
```
Comment (badb) [inline tests/golem/docker/test_blender_job.py:44]: On windows it's represented as '\golem\resources\cube.blend`, therefore this test fails.

**Review by badb — CHANGES_REQUESTED** [review]

**Review by badb — COMMENTED** [review]
Two more questions. 
Can you remove gpu version (we're not going to use it now)? 
Can  you make benchmark lighter? Like 2-3 times lighter - you can either decrease the reslution or spp in blender scene.

**Comment by jiivan** [comment]
I decreased resolution from `1920x1080` to `480x270`. I also decreased sampling->samples->render from 35 to 20. Overall it changed my performance from `~160` to `~400`.

**Review by badb — APPROVED** [review]

**Pull request merged**